### PR TITLE
refactor: expand

### PR DESCRIPTION
**Pull request by francescoopiccoli**

### CODE REVIEW

File: data_generation/expand.py [hunk 1]
```diff
@@ -27,28 +27,51 @@ def get_args():
     )
     # if file is moved in another directory level relative to the root (currently in root/data_generation), this needs to be changed
     root_directory = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    parser.add_argument("-d", "--data_path", default=root_directory + "/Data/train")
-    parser.add_argument("-s", "--save_dir", default=root_directory + "/Data/expand")
+    parser.add_argument("-d", "--data_path", default=root_directory + "/Data")
+    parser.add_argument("--dataset", choices=['train', 'val', 'test'], default='train')
+    parser.add_argument("-s", "--save_dir", default=root_directory + "/Data")
     args = parser.parse_args()
```
Comment (AlexDeLos) [inline data_generation/expand.py:34]: Do you have no control over the amount of graphs that are created from every graph?

Comment (AlexDeLos) [reply data_generation/expand.py:34]: added it myself

**Review by AlexDeLos — APPROVED** [review]
if the changes look good to you they are good for me

File: data_generation/expand.py [hunk 2]
```diff
@@ -27,49 +27,88 @@ def get_args():
...
-        for _ in range(int(len(graph.bus) / 4)):
-            new_graph = deepcopy(graph)
-            for i in graph.gen.index:
-                new_graph.gen.loc[i, 'p_mw'] = np.random.normal(graph.gen.loc[i, 'p_mw'], math.sqrt(abs(graph.gen.loc[i, 'p_mw'])))
-            for i in graph.load.index:
-                new_graph.load.loc[i, 'p_mw'] = np.random.normal(graph.load.loc[i, 'p_mw'], math.sqrt(abs(graph.load.loc[i, 'p_mw'])))
-                new_graph.load.loc[i, 'q_mvar'] = np.random.normal(graph.load.loc[i, 'q_mvar'], math.sqrt(abs(graph.load.loc[i, 'q_mvar'])))
+        for _ in range(args.num_networks):
+            graph = pp.from_json(f"{p}/x/{g}")
+            for _ in range(int(len(graph.bus) / 4)):
```
Comment (francescoopiccoli) [inline data_generation/expand.py]: I think the 2nd for loop `for _ in range(int(len(graph.bus) / 4)):` should be removed and `graph = pp.from_json(f"{p}/x/{g}")` would be better outside of the` for _ in range(args.num_networks)`. With the current two for loops we have here, we generate `num_networks * int(len(graph.bus) / 4)` varied graphs for each graph

Comment (AlexDeLos) [reply data_generation/expand.py]: changes discussed have been added

**Pull request merged**